product/tasks.md

```diff
@@ -71,13 +71,13 @@ The `x:size/<value>` labels describe the expected amount of work for a contribut
 
 The `x:type/<value>` labels describe the type of work the contributor will be engaged in.
 
-| Tag              | Description                                                    |
-| ---------------- | -------------------------------------------------------------- |
-| `x:type/ci`      | Work on Continuous Integration (e.g. GitHub Actions workflows) |
-| `x:type/coding`  | Work on code that will run in production                       |
-| `x:type/content` | Work on content (e.g. exercises, concepts)                     |
-| `x:type/docker`  | Work on Dockerfiles                                            |
-| `x:type/docs`    | Work on Documentation                                          |
+| Tag              | Description                                                       |
+| ---------------- | ----------------------------------------------------------------- |
+| `x:type/ci`      | Work on Continuous Integration (e.g. GitHub Actions workflows)    |
+| `x:type/coding`  | Work on code that is not part of the content (e.g. track tooling) |
```

Comment:
```suggestion
| `x:type/coding`  | Work on code that is not part of the content (e.g. is a track tooling) |
```
Could be understood _"e.g. is NOT a track tooling"_, because `not` has been used before parenthesis.

Reply:
I don't know, adding a verb inside the parenthesis makes it sound too complicated to me 🤔. IMO anyone who understands what track tooling is will also know that it is not content. With the article it also sounds somehow wrong, is it possible that `track tooling` is uncountable?

Reply:
Seems you're right about uncountability/article. Regarding a verb, for me it still makes things clearer. I'm not sure I have anything more to add.
```suggestion
| `x:type/coding`  | Work on code that is not part of the content (e.g. is track tooling) |
```